torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -139,12 +114,8 @@ def resize_image_tensor(
         # This is a perf hack to avoid slow channels_last upsample code path
         # Related issue: https://github.com/pytorch/pytorch/issues/83840
         # We are transforming (N, 1, H, W) into (N, 2, H, W) to force to take channels_first path
-        if (
-            image.is_contiguous()
-            and is_channels_last_contiguous_2d(image)
-            and interpolation == InterpolationMode.NEAREST
-        ):
-            # Code is copied from _FT.resize
+        if image.shape[1] == 1 and interpolation == InterpolationMode.NEAREST:
```

Comment:
For arbitrary dimensions?
```suggestion
        if image.shape[-3] == 1 and interpolation == InterpolationMode.NEAREST:
```

Reply:
It is not arbitrary anymore as we viewed it into 4d tensor

Reply:
My bad, you are right.